.github/workflows/continuous-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Deployment"
+
+on:
+    push:
+        tags:
+            - "**"
+
+jobs:
+    release:
+        name: "Release"
+
+        runs-on: "ubuntu-latest"
+
+        steps:
+            - name: "Determine tag"
+              id: "determine-tag"
+              run: "echo \"::set-output name=tag::${GITHUB_REF#refs/tags/}\""
+
+            - name: "Create release"
+              id: "create-release"
+              uses: "actions/create-release@v1"
+              env:
+                  GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+              with:
+                  draft: false
+                  prerelease: false
+                  release_name: "${{ steps.determine-tag.outputs.tag }}"
+                  tag_name: "${{ steps.determine-tag.outputs.tag }}"
+
+            - name: "Upload PHAR"
+              uses: "actions/upload-release-asset@v1.0.1"
+              env:
+                  GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+              with:
+                  upload_url: "${{ steps.create-release.outputs.upload_url }}" # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
+                  asset_path: "./rector.phar"
+                  asset_name: "rector.phar"
+                  asset_content_type: "application/octet-stream"
```

Comment:
There are already is local `rector.phar` pushed from `rector/rector`repository

What is this needed for?

Reply:
This is just to publish the pushed `rector.phar` as an asset of the release.

Reply:
Ah, I see.

Could you add there `rector` as well, for the consistency.

Reply:
I don't think it's a good idea. They are the same files. I think it'll just confuse our users.

Reply:
The same way it would confuse `rector` and `rector.phar` in src.

Some people might depend on it, since it's the rule in main repository.

Better solve it once for all resources in consistent way, than once this way and one other way.

Reply:
Done.